src/mame/drivers/rx78.cpp

```diff
@@ -25,14 +25,40 @@ Driver by Angelo Salese & Robbbert.
 - To stop a cmt load, press STOP + SHIFT keys
 
 Known programs:
```

Comment:
If the software list contains a list of all known software, it’s better to remove this list altogether.  Keeping it here just risks having it get out of sync.